cmd/argocd-util/commands/test_clusterrole.yaml

```diff
@@ -0,0 +1,787 @@
+aggregationRule:
```

Comment:
Please move file into `cmd/argocd-util/commands/projects/testdata` directory to be consistent with rest of the code.